app/assets/stylesheets/common/communicarts.scss

```diff
@@ -80,7 +80,19 @@ table td span.tag-values {
   font-weight: normal;
 }
 
+td.amount {
```

Comment:
sometimes github PR view is weird-- shows this file as existing in overview but I see in file view that it has beeb renamed. thanks for the update!